lib/uwb/include/uwb/UwbDevice.hxx

```diff
@@ -51,6 +51,12 @@ public:
     void
     Reset();
 
+    /**
+     * @brief Initializes a new UWB device
+     */
+    virtual bool
+    Initialize();
```

Comment:
Looks like this doxygen comment needs updating to account for `bool` return type.

Reply:
Yes, good catch!